example/ck_tile/01_fmha/fmha_bwd.cpp

```diff
@@ -265,6 +265,9 @@ bool run(const ck_tile::ArgParser& arg_parser)
         (mode == mode_enum::batch ? seqlen_q : seqstart_q_host.back());
     const ck_tile::index_t shape_seqlen_k =
         (mode == mode_enum::batch ? seqlen_k : seqstart_k_host.back());
+    const ck_tile::index_t kN0 = (hdim_q > 32 & hdim_q <= 128) ? 128 : 64;
```

Comment:
One question, so the logic to determine `kN0`  here should be consistent with the one to determine the `kN0` in the TileShapes, right?

The good way is not to expose the logic here, but let the kernel to provide a interface "GetWorkspaceSize()`.